src/Playwright/Core/Page.cs

```diff
@@ -449,6 +449,10 @@ public async Task CloseAsync(PageCloseOptions options = default)
         {
             try
             {
+                if (IsClosed)
```

Comment:
Why is this check here? You can still call CloseAsync twice, even with this check.

Reply:
If this check isn't put there, it'll throw when calling CloseAsync. Doing something similar to Playwright Java - https://github.com/microsoft/playwright-java/blob/main/playwright/src/main/java/com/microsoft/playwright/impl/PageImpl.java#L542